cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/task/RunTestsTask.java

```diff
@@ -349,6 +349,18 @@ private int runTestSuit(Path jsonPath, BuildContext buildContext, HashSet<Path>
 
         String jacocoAgentJarPath = Paths.get(System.getProperty(BALLERINA_HOME)).resolve(BALLERINA_HOME_BRE)
                 .resolve(BALLERINA_HOME_LIB).resolve(TesterinaConstants.AGENT_FILE_NAME).toString();
+
+        Path jacocoCoreJarPath =  Paths.get(System.getProperty(BALLERINA_HOME)).resolve(BALLERINA_HOME_BRE)
```

Comment:
Shall we move this set of dependency loading to a separate function?

Reply:
Moved to separate functions